src/Symfony/Component/VarDumper/Caster/ConstStub.php

```diff
@@ -30,4 +30,27 @@ public function __toString()
     {
         return (string) $this->value;
     }
+
+    /**
+     * Creates ConstStub from bitfield flag.
+     *
+     * @param int    $value  The value of bitfield flag
+     * @param string $prefix The prefix filter applied on constant names
+     *
+     * @return self
+     */
+    public static function fromFlag($value, $prefix)
```

Comment:
We have a different pov... note im not sure what's best. Im just questioning this.. so lets see what others think. Otherwise keep it as-is ;-)

> Mmh, if I use E_SOMETHING_UNRELATED, I would like see it in the dump.

> a new E_ constant like define('E_CUSTOM', 32768), it will not be dumped in your case

We should really consider what we care about; which context adds more value;

Yours:
Try to show exactly what the developer passed

Mine:
Show the _actual_ severity for a error exception; where either `E_SOMETHING_UNRELATED` or `E_CUSTOM` is not related; it's just `int(2)` (known as `E_WARNING`) and `int(32768)`